babel.config.js

```diff
@@ -15,14 +20,14 @@ module.exports = function( api ) {
 				'inline-react-svg',
 				{
 					svgo: {
-						plugins: [
+						plugins: extendDefaultPlugins( [
```

Comment:
The updated `babel-plugin-inline-react-svg` dependency now makes of `svgo` v2, which has updated the syntax in how its plugins should be [configured](https://github.com/svg/svgo/blob/v2.1.0/README.md?rgh-link-date=2021-02-24T08%3A53%3A38Z#configuration). I've made the adjustments in 5aacae1dc804117acf00ceac7cf9ba58be401294.